Module.yaml

```diff
@@ -27,34 +25,23 @@ envs:                                   # Array of environment variables that wi
     key: SLAVE_ADDR                      # key to set in the environment variables.
-    description: I2c slave address       # Description/helper text for the field.
+    description: I2c slave address (in decimal)     # Description/helper text for the field.
     value: ''                            # Value to be set in the environment variables.
     type: 'number'                       # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
-    options: []                          # Options for the field if it is a select or multiple select. An array of string options
-    dependencies: []                     # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
-  - name: Polling Period                 # name of the field to be displayed on the UI.
-    key: PERIOD                          # key to set in the environment variables.
-    description: The I2C interface read data every choosen period (in seconds)      # Description/helper text for the field.
-    value: ''                           # Value to be set in the environment variables.
-    type: 'number'                      # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
-    options: []                         # Options for the field if it is a select or multiple select. An array of string options
-    dependencies: []                    # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
-  - name: Offset (in decimal)           # name of the field to be displayed on the UI.
+  - name: Offset          # name of the field to be displayed on the UI.
     key: OFFSET                         # key to set in the environment variables.
-    description: Offset of a value (used in case a sensor provide multiple values)     # Description/helper text for the field.
+    description: Offset in case the sensor has more than one registers available (in decimal)       # Description/helper text for the field.
     value: 0                            # Value to be set in the environment variables.
     type: number                        # Type of the field. Possible values are: ['text', 'select', 'boolean', 'date', 'time', 'dateTime-local', 'number', 'multiple', 'CSV', 'file']
-    options: []                         # Options for the field if it is a select or multiple select. An array of string options
-    dependencies: []                    # List of dependencies for the field. This field is only displayed if all the dependencies are met. If
-ports:                                  # Array of ports that will be exposed by the module.
-  - container: 8080                     # Port number from the container.
-    host: 8080                          # Port number to map on the host.
-tags:
```

Comment:
did you accidentally delete this line?

Reply:
nope, I thought it doesn't matter to keep it because there is no mounting device anymore.

Reply:
I'm talking about `tags:`